llvm/test/tools/llvm-objcopy/ELF/gap-fill-order.test

```diff
@@ -0,0 +1,34 @@
+# RUN: yaml2obj %s >%t
```

Comment:
Nit: most of our newer tests use the `-o` option instead of output redirection (which has potential issues when copy-pasted into certain terminals, e.g. PowerShell).

Reply:
I change it to use the `-o` option now.